Makefile

```diff
@@ -13,6 +14,16 @@ endif
 
 all: manager
 
+dev-up: dev-cluster dev-deploy
+dev-down: stop
+	kind delete cluster orkestra
```

Comment:
Does this command work? I thought you needed to specify `--name`?

Reply:
yes - I need to push a new commit to this PR with the fixes and some better targets but I am hitting some snags. Will update this PR soon